### DISCUSSION TITLE
Copilot .vimrc Tab Completion Not Working

### DISCUSSION THREAD

**Discussion by xeb** [community/community#43076] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi all,

After installing [GitHub Copilot.vim](https://github.com/github/copilot.vim) for both vim and nvim, I was **unable to accept a suggestion**. While I was authenticated and `:Copilot status` showed everything was enabled & online, I could not accept a suggestion. I was also unable to re-bind my accept suggestion command to a new key (e.g. `<C J>`). Nothing worked

After a bit of trial & error, I found the problem. Whenever I set **`set paste`** in my `~/.vimrc` tab completion stops working. If I remove **`set paste`** then both vim and nvim work just fine with Copilot.

Does anyone know why? Is there a valid work around?

Thanks!

**Comment by tpope** [comment, marked as answer]
`:set paste` disables all maps. It's not something that should be set permanently.

**Reply by hotelnetwork** [reply]
Just checked on my system, not working.. Whoever wrote Copilot needs to check their hypotenuse.

**Reply by tpope** [reply]
Maybe check the hypotenuse of your tab map with `:verbose imap <Tab>`.
